src/isolate.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import pandas as pd
+import numpy as np
+
+from scipy.io import wavfile
+from scipy import signal, interpolate
+from matplotlib import pyplot as plt
+from util import *
+
+
+# Read the audio file
+data_dir = "training_data"
```

Comment:
Can we put this whole thing into a main()?

Reply:
I added it to a function taking a label and path parameter instead since you will be needing to pass these from main.py,  and I don't think that's supported with __main__